src/bson.c

```diff
@@ -763,23 +772,30 @@ PHONGO_API void zval_to_bson(zval *data, php_phongo_bson_flags_t flags, bson_t *
 
 	switch(Z_TYPE_P(data)) {
 		case IS_OBJECT:
-			if (instanceof_function(Z_OBJCE_P(data), php_phongo_persistable_ce TSRMLS_CC)) {
+			if (instanceof_function(Z_OBJCE_P(data), php_phongo_serializable_ce TSRMLS_CC)) {
```

Comment:
For PHPC-274: this swap of `php_phongo_persistable_ce` to `php_phongo_serializable_ce` is the relevant change for satisfying the non-embedded test cases in `bson-encode-002.phpt` (the embedded test cases already passed before this).